client/mining_common.c

```diff
@@ -53,10 +53,7 @@ static int crypt_start(void)
 miner_addr - address of the miner, if specified */
 int xdag_initialize_mining(int pool_on, const char *pool_arg, const char *miner_address)
```

Comment:
It makes sense to remove this parameter and use global variable.

Reply:
Good point. It's make sense to use global variable.

Reply:
Good idea! It makes sense. I will change it.